src/llama.cpp

```diff
@@ -426,6 +427,7 @@ static const std::map<llm_kv, const char *> LLM_KV_NAMES = {
     { LLM_KV_SSM_INNER_SIZE,                "%s.ssm.inner_size"     },
     { LLM_KV_SSM_STATE_SIZE,                "%s.ssm.state_size"     },
     { LLM_KV_SSM_TIME_STEP_RANK,            "%s.ssm.time_step_rank" },
+    { LLM_KV_SSM_B_DT_RMS,                  "%s.ssm.b_dt_rms" },
```

Comment:
Not sure about the name. Naming is hard and then we're stuck with what was chosen.


```suggestion
    { LLM_KV_SSM_DT_B_C_RMS,                  "%s.ssm.dt_b_c_rms" },
```
Note that I'm not sure about this suggestion either, but at least it's *slightly* more representative of its effect.

If changed, this should also be updated in `gguf-py`.

Reply:
Makes sense, let me accept your suggestion and replace it all over the place